test/testutil/buffer_generator.hpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_TESTUTILS_BUFFER_GENERATOR_HPP
+#define CPP_FILECOIN_TESTUTILS_BUFFER_GENERATOR_HPP
+
+#include <libp2p/crypto/random_generator/boost_generator.hpp>
+
+#include "common/buffer.hpp"
+
+using fc::common::Buffer;
+using libp2p::crypto::random::BoostRandomGenerator;
+using libp2p::crypto::random::CSPRNG;
+
+namespace testutils {
+
+  /**
+   * @brief Random buffer generator
+   */
+  class BufferGenerator {
+   public:
+    static constexpr size_t kDefaultBufferLength = 32;
+
+    /**
+     * Generate random buffer if size length
+     * @param size - length of generated buffer
+     * @return random bytes buffer
+     */
+    Buffer makeRandomBuffer(size_t size = kDefaultBufferLength) {
```

Comment:
```suggestion
    Buffer makeRandomBuffer(size_t size) {
```

that default size was for cid, there is no default size for all purposes.